src/Raven.Server/Documents/Replication/DocumentReplicationLoader.cs

```diff
@@ -540,7 +540,7 @@ private int ReplicatedPast(long etag)
             int count = 0;
             foreach (var destination in _outgoing)
             {
-                if (destination._lastSentDocumentEtag >= etag)
+                if (destination._lastNoSystemDocumentEtag >= etag)
```

Comment:
Don't know about the name.
Because this etag is the last accepted etag but without the count of the system.
Will call him lastReceivedDocumentEtag